### ISSUE TITLE
Compilation failed on OS X

### ISSUE THREAD

**Issue by nuboat** [nunit/dotnet-test-nunit#66]
I have tried simple project.json on OS X,
dotnet restore run without problemt but got failed when run dotnet test
```
Project YoWeb (.NETCoreApp,Version=v1.0) will be compiled because inputs were modified
Compiling YoWeb for .NETCoreApp,Version=v1.0
/usr/local/share/dotnet/dotnet compile-csc @~/Projects/YoWeb/obj/Debug/netcoreapp1.0/dotnet-compile.rsp returned Exit Code 1
~/Projects/YoWeb/test/YoWeb/Facade/HelloWorldTest.cs(1,7): error CS0246: The type or namespace name 'NUnit' could not be found (are you missing a using directive or an assembly reference?)
~/Projects/YoWeb/test/YoWeb/Facade/HelloWorldTest.cs(5,6): error CS0246: The type or namespace name 'TestFixtureAttribute' could not be found (are you missing a using directive or an assembly reference?)
~/Projects/YoWeb/test/YoWeb/Facade/HelloWorldTest.cs(5,6): error CS0246: The type or namespace name 'TestFixture' could not be found (are you missing a using directive or an assembly reference?)
~/Projects/YoWeb/test/YoWeb/Facade/HelloWorldTest.cs(10,10): error CS0246: The type or namespace name 'TestAttribute' could not be found (are you missing a using directive or an assembly reference?)
~/Projects/YoWeb/test/YoWeb/Facade/HelloWorldTest.cs(10,10): error CS0246: The type or namespace name 'Test' could not be found (are you missing a using directive or an assembly reference?)

Compilation failed.
    0 Warning(s)
    5 Error(s)

Time elapsed 00:00:01.7011026

SUMMARY: Total: 1 targets, Passed: 0, Failed: 1.
```

**Comment by jasonwilliams200OK** [comment]
I have updated the README example: https://github.com/nunit/dotnet-test-nunit/blob/master/README.md#projectjson. You need to add NUnit as a dependency in project.json, then specify `using NUnit.Framework;` in your code file.

**Comment by jasonwilliams200OK** [comment]
Now that beta-2 is released, update the version to `"3.4.0-beta-2"`.

Please follow the new usage instructions in README: https://github.com/nunit/dotnet-test-nunit/blob/master/README.md#usage, and reopen the issue if you are still having problems.
